tests/jni_api.rs

```diff
@@ -143,3 +149,54 @@ pub fn with_local_frame_pending_exception() {
 
     env.exception_clear().unwrap();
 }
+
+#[test]
+pub fn call_static_method_ok() {
+    let env = attach_current_thread();
+
+    let x = JValue::from(-10);
+    let val: jint = env.call_static_method(CLASS_MATH, METHOD_MATH_ABS, SIG_MATH_ABS, &[x])
+        .expect("JNIEnv#call_static_method_unsafe should return JValue").i().unwrap();
+
+    assert_eq!(val, 10);
+}
+
+#[test]
+pub fn call_static_method_throws() {
+    let env = attach_current_thread();
+
+    let x = JValue::Long(4_000_000_000);
+    env.call_static_method(CLASS_MATH, METHOD_MATH_TO_INT, SIG_MATH_TO_INT, &[x])
+        .expect_err("JNIEnv#call_static_method_unsafe should return error");
```

Comment:
Can we check that the error is JavaException?

https://github.com/jni-rs/jni-rs/blob/fec275ffa42353778ad34ecb903c9b45336517b9/src/wrapper/errors.rs#L32-L34